en/messages.json

```diff
@@ -0,0 +1,126 @@
+    },
+    "firstRunP": {
+        "message": "You’ve successfully installed the Firefox Relay add-on. To get started, sign in to Relay with your Firefox account."
+    },
+    "firstRunCollectDataLabel": {
+        "message": "Allow Mozilla to collect information about how you use this extension."
+    },
+    "disableDataCollection": {
+        "message": "Disable data collection"
+    },
+    "firstRunPrefP": {
+        "message": "Collecting interaction data when you do things like click the Relay icon in your toolbar or create a new alias helps us learn what is working well and where we can improve. You can always turn this off in the Settings panel."
+    },
+    "firstRunSignInButton": {
+        "message": "Sign Up / In"
+    },
+    "popupSettingsTitle": {
+        "message": "Settings"
+    },
+    "popupSettingsReturn": {
+        "message": "Return to main panel"
+    },
+    "popupSettingsShowIcon": {
+        "message": "Show Relay icon in email fields on websites"
+    },
+    "popupSettingsCollectData": {
+        "message": "Allow Mozilla to collect interaction data"
+    },
+    "popupSettingsLeaveFeedback": {
+        "message": "Leave Feedback"
+    },
+    "popupSettingsPrivacy": {
+        "message": "Privacy"
+    },
+    "popupSettingsTermsOfService": {
+        "message": "Terms of Service"
+    },
+    "popupSignUpPanelWelcome": {
+        "message": "Welcome"
+    },
+    "popupSignUpPanelIntro": {
+        "message": "Get started by signing in to Relay with your Firefox account."
+    },
+    "popupSignUpPanelContinue": {
+        "message": "Continue"
+    },
+    "popupRemainingAliases": {
+        "message": "You have $REMAINING$/$MAX$ remaining aliases",
+        "description": "Tells the user how many more aliases they can make.",
+        "placeholders": {
+            "remaining": {
+                "content": "$1"
+            },
+            "max": {
+                "content": "$2"
+            }
+        }
+    },
+    "popupUnlimitedAliases": {
+        "message": "You have created $COUNT$ aliases",
+        "description": "Tells the aliases these premium user has created.",
+        "placeholders": {
+            "count": {
+                "content": "$1"
+            }
+        }
+    },
+    "popupOnboardingShowPrevious": {
+        "message": "Show previous panel"
+    },
+    "popupOnboardingShowNext": {
+        "message": "Go to next panel"
+    },
+    "ManageAllAliases": {
+        "message": "Manage All Aliases"
+    },
+    "popupOnboardingPanel1Body": {
+        "message": "When the Firefox Relay icon appears on a website, select it to generate a new alias."
+    },
+    "popupOnboardingPanel2Headline": {
+        "message": "Don't see the icon?"
+    },
+    "popupOnboardingPanel2Body": {
+        "message": "Right-click (Windows) or Control-click (macOS) on form fields to access the context menu and generate an alias from there."
```

Comment:
If we want to re-write this message to indicate we support all the platforms, let's do.